src/include/txn_inline.h

```diff
@@ -389,6 +389,8 @@ __wt_txn_modify(WT_SESSION_IMPL *session, WT_UPDATE *upd)
         if (F_ISSET(txn, WT_TXN_IGNORE_PREPARE))
             WT_RET_MSG(
               session, ENOTSUP, "Transactions with ignore_prepare=true cannot perform updates");
+        __wt_verbose_debug(
+          session, WT_VERB_TRANSACTION, "Rollback reason: %s", "Write in read-only txn");
```

Comment:
np: Same here about string formatting

Reply:
Same, forgot this one, cheers!

Reply:
Please refer to the first comment related to this change.